### PR TITLE
chore(FEC-9476): added unit tests for Progressive sources

### PR DESCRIPTION
**Pull request by shalommeoded**
### Description of the Changes

Added unit test for progressive sources
SOLVED FEC-9476
### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [x] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: test/src/k-provider/ovp/playback-sources-data.js [hunk 1]
```diff
@@ -76,6 +76,34 @@ export const kalturaDashSource = new KalturaPlaybackSource({
...
+export const kalturaProgressiveSourceSecured = new KalturaPlaybackSource({
+  drm: [],
+  format: 'url',
+  deliveryProfileId: 19951,
+  url: 'myUrl',
+  protocols: 'https',
+  flavorIds: '0_m131krws,0_5407xm9j,0_xcrwyk2n'
+});
+
+export const kalturaProgressive = new KalturaPlaybackSource({
```
Comment (RoyBregman) [inline test/src/k-provider/ovp/playback-sources-data.js]:
```suggestion
export const kalturaProgressiveMultiProtocol = new KalturaPlaybackSource({
```

File: test/src/k-provider/ovp/provider-parser.spec.js [hunk 2]
```diff
@@ -35,6 +38,38 @@ describe('provider parser', function() {
...
+        [kalturaProgressiveSourceNotSecured, kalturaProgressiveSourceSecured],
+        'myKS',
+        1234,
+        1234,
+        {
+          id: '1_938734'
+        },
+        context
+      );
+      progressiveSource.should.exist;
```
Comment (RoyBregman) [inline test/src/k-provider/ovp/provider-parser.spec.js]: don't we want to check the the selected source is the one you would expect? should equal protocol or deliveryProfileId?

File: test/src/k-provider/ovp/provider-parser.spec.js [hunk 3]
```diff
@@ -35,6 +38,38 @@ describe('provider parser', function() {
...
+        [kalturaProgressive],
+        'myKS',
+        1234,
+        1234,
+        {
+          id: '1_938734'
+        },
+        context
+      );
+      progressiveSource.should.exist;
```
Comment (RoyBregman) [inline test/src/k-provider/ovp/provider-parser.spec.js]: same here - don't we want to check the the selected source is the one you would expect? should equal protocol or deliveryProfileId?

File: test/src/k-provider/ovp/provider-parser.spec.js [hunk 4]
```diff
@@ -35,6 +39,46 @@ describe('provider parser', function() {
...
+      progressiveSource.progressive[0].id.should.equal('0_5407xm9j19951,url');
+      progressiveSource.progressive[0].url.should.equal(
+        'https://cdnapisec.kaltura.com/p/1234/sp/123400/playManifest/entryId/1_938734/protocol/https/format/url/flavorIds/0_5407xm9j/ks/myKS/a.mp4?uiConfId=1234'
+      );
```
Comment (RoyBregman) [inline test/src/k-provider/ovp/provider-parser.spec.js]: i think the test in line 57 is enough but if you wish to check also the url please use the data from the MediaConfig and not put that hard coded as someone might want to change it some day in the media.js and this test will fail...

File: test/src/k-provider/ovp/provider-parser.spec.js [hunk 5]
```diff
@@ -35,6 +39,46 @@ describe('provider parser', function() {
...
+        {
+          id: '1_938734'
+        },
+        context
+      );
+      progressiveSource.should.exist;
+      progressiveSource.progressive[0].id.should.equal('0_5407xm9j19961,url');
+      progressiveSource.progressive[0].url.should.equal(
+        'https://cdnapisec.kaltura.com/p/1234/sp/123400/playManifest/entryId/1_938734/protocol/https/format/url/flavorIds/0_5407xm9j/ks/myKS/a.mp4?uiConfId=1234'
+      );
```
Comment (RoyBregman) [inline test/src/k-provider/ovp/provider-parser.spec.js]: same here (look my comment above)

Comment (shalommeoded) [reply test/src/k-provider/ovp/provider-parser.spec.js]: done

**Pull request merged**